### PR TITLE
Login to TU / Schedule Room

### PR DESCRIPTION
**Pull request by ccho-0508**
Run main.py :
- Opens new window (switch back to IDE command line to input information)
- Takes in TU credentials to login
- Provide time, date, and room to make a reservation
- Switch to window opened to watch selenium work
- Exits if : login is incorrect, time/date/room given is in incorrect format or not available 
- WILL NOT SCHEDULE ROOM : Just clicks on the time slot and gets you to the confirmation page - Logs out in the end

### CODE REVIEW

**Comment by bobby-palko** [comment]
I pulled this down and tested it out. Works great! I did run into an issue with 2FA; the current wait threshold isn't quick enough for me to complete the 2FA processes, but it also will ask every time it logs in — something to think about.

For our first demo we may want to bump up the wait time (I'll mark it) in case the person testing it also has 2FA on their account.

File: proofOC/main.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    driver.get("https://tuportal.temple.edu/")
+    driver.maximize_window()
+
+    driver.find_element(By.XPATH, "//input[@id='username']").send_keys(user)
+    driver.find_element(By.XPATH, "//input[@id='password']").send_keys(passW)
+    driver.find_element(By.XPATH, "//button[@class='btn btn-default btn-login']").click()
+
+    # If login fails check for error
+    try:
+        error = WebDriverWait(driver, 5).until(EC.presence_of_element_located((By.XPATH, "//span[@class='logout']")))
```
Comment (bobby-palko) [inline proofOC/main.py:31]: bumping this up to something like 30 or 60 should be sufficient

**Comment by ccho-0508** [comment]
You can now specify if you want to reserve a large room and the wait time to check if the login was successful is now 30 seconds.

**Comment by bobby-palko** [comment]
looks good to me for the PoC. once you're good to merge this i'll give it the ✅

**Review by bobby-palko — APPROVED** [review]
looks good!

**Pull request merged**